main/src/ca/uwaterloo/flix/language/ast/ParsedAst.scala

```diff
@@ -574,6 +574,15 @@ object ParsedAst {
       */
     case class Match(sp1: SourcePosition, exp: ParsedAst.Expression, rules: Seq[ParsedAst.MatchRule], sp2: SourcePosition) extends ParsedAst.Expression
 
+    /**
+      * Select Expression (select first channel expression).
+      *
+      * @param sp1   the position of the first character in the expression.
+      * @param rules the rules of the select channel.
+      * @param sp2   the position of the last character in the expression.
+      */
+    case class Select(sp1: SourcePosition, rules: Seq[ParsedAst.SelectRule], sp2: SourcePosition) extends ParsedAst.Expression
```

Comment:
Looks good! Maybe rename it to `SelectChannel` to follow the same style as the other AST fragments.